.github/workflows/deploy-wheels.yml

```diff
@@ -41,45 +23,45 @@ jobs:
         uses: actions/setup-python@v3
         with:
           python-version: 3.9
-        if: ${{ matrix.config.os != 'macos-14' }}
+        if: ${{ matrix.os != 'macos-14' && matrix.os != 'macos-latest' }}
       - name: Install LLVM
         uses: ./.github/actions/install-llvm
         with:
           version: "14"
-          os: ${{ matrix.config.os }}
+          os: ${{ matrix.os }}
           directory: ${{ github.workspace }}/target/llvm
-          target: ${{ matrix.config.target }}
-      - name: "Build ${{ matrix.config.target }}"
+          target: "pypi-build"
+      - name: "Run build script"
         run: |
           pwd
-          ./build.ps1 -t ${{ matrix.config.target }}
+          ./build.ps1 -t "pypi-build"
         shell: pwsh
       - name: Artifacts - manylinux
         uses: actions/upload-artifact@v4
         with:
           name: rasqal-manylinux-artifacts
           path: src/target/wheels/*manylinux_2_*.whl
-        if: ${{ matrix.config.os == 'ubuntu-20.04' }}
+        if: ${{ matrix.os == 'ubuntu-20.04' || matrix.os == 'ubuntu-latest' }}
       - name: Artifacts
         uses: actions/upload-artifact@v4
         with:
-          name: rasqal-${{ matrix.config.os }}-artifacts
+          name: rasqal-${{ matrix.os }}-artifacts
           path: src/target/wheels/*
-        if: ${{ matrix.config.os != 'ubuntu-20.04' }}
+        if: ${{ matrix.os != 'ubuntu-20.04' && matrix.os != 'ubuntu-latest' }}
 
-  publish:
-    runs-on: ubuntu-latest
-    needs: [ deploy ]
-    permissions:
-      id-token: write
-    steps:
-      - uses: actions/download-artifact@v4
-        with:
-          pattern: rasqal-*
-          merge-multiple: true
-          path: dist/
-
-      - name: Publish to PyPi
-        uses: pypa/gh-action-pypi-publish@release/v1
-        with:
-          packages-dir: dist/
\ No newline at end of file
+#  publish:
```

Comment:
how come commented out?

Reply:
Testing, don't want to deploy. I haven't set up different environments yet so it doesn't auto-deploy to test pypi.